Review.java

```diff
@@ -165,4 +165,24 @@ public static String randomAdjective()
       return randomNegativeAdj();
     }
   }
+   public static int starRating(String fileName);
+   {

```

Comment:
i would add comments explaining what you are doing. otherwise it looks good

Reply:
OK thank you.